static/src/javascripts/projects/common/modules/identity/account-profile.js

```diff
@@ -96,8 +104,58 @@ define([
         }
     };
 
+    accountProfile.prototype.avatarUploadByGAE = function (avatarForm) {
```

Comment:
I think we usually use https://github.com/ded/reqwest for ajax stuff although someone else might be along soon to give a more authoritative view on that